thriftclient/client_pool.go

```diff
@@ -61,16 +61,6 @@ type ClientPoolConfig struct {
 	PoolGaugeInterval time.Duration
 }
 
-// getMetricLabels returns c.MetricsLabels as a slice of strings, appropriate
-// for passing to a metrics client.
-func (c ClientPoolConfig) getMetricLabels() []string {
-	labels := make([]string, len(c.MetricsLabels)*2)
```

Comment:
We had a bug here, this was initializing a list of length `len(c.MetricsLabels)*2` full of empty strings and then appending the actual labels to that list.

Reply:
🤦‍♂